src/libs/websocket.ts

```diff
@@ -0,0 +1,14 @@
+import { Accessor, Context, createContext, Setter } from "solid-js";
+
+// FIXME: Don't pass message signal through the websocket context data.
+type WebsocketContextData = {
+  websocket: WebSocket;
+  messages: Accessor<Message[]>;
+  setMessages: Setter<Message[]>;
+};
+
+export const WebsocketContext: Context<WebsocketContextData | undefined> = createContext(undefined);
+
+export function sendWSMessage(ws: WebSocket, message: string) {
```

Comment:
If we can get websockets to be stored globally, i.e through [stores](https://docs.solidjs.com/concepts/stores) or context, we can use this module as a means of communicating with them directly and revoke the need for this function.

The end goal is to essentially store all websocket references within his module, and to manipulate them either from a variable or other functions with more catered names, i.e `sendChatMessage(...)` over the (slightly) more verbose `sendWSMessage(...)`.

Thoughts?

Reply:
I'll look into it